ui/src/pages/custody/Account.tsx

```diff
@@ -221,7 +209,7 @@ const AccountComponent: React.FC<RouteComponentProps & ServicePageProps<Service>
     <>
       <BackButton prevPageLabel="Wallet" prevPagePath={paths.app.custody.assets} />
       <InputDialog {...transferDialogProps} isModal />
-      <InputDialog {...creditDialogProps} isModal />
+      {/*<InputDialog {...creditDialogProps} isModal />*/}
```

Comment:
did u mean to leave this commented out?

Reply:
No I didn't mean to leave this in! Thanks for pointing this out!! 👀